.github/workflows/pr-tests.yml

```diff
@@ -23,220 +23,220 @@ jobs:
     runs-on: warp-ubuntu-2404-x64-8x
 
     steps:
-    - uses: actions/checkout@v4
-      with:
-        submodules: recursive
-    - name: ⚡ Restore rust cache
-      id: cache
-      uses: WarpBuilds/cache/restore@v1
-      with:
-        path: |
-          ~/.cargo/registry/index/
-          ~/.cargo/registry/cache/
-          ~/.cargo/git/db/
-          target/
-          Cargo.lock
-        key: ${{ runner.os }}-cargo-pr-tests
-    - name: Date of the restored cache
-      run: cat target/cache-build-date.txt
-      continue-on-error: true
+      - uses: actions/checkout@v4
```

Comment:
this was all fmt, no idea why

Reply:
you mean `cargo fmt` ? Would it mean it looks at yml files as well?